src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -156,5 +165,11 @@ export default React.memo(
                 key: ONYXKEYS.COLLECTION.POLICY,
             },
         }),
+        withOnyx({
+            parentReportActions: {
+                key: ({fullReport}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${fullReport.parentReportID}`,
+                canEvict: false,
+            },
```

Comment:
This will make the option row re-render whenever any report action change (and not just the parent report action). This can be fixed in two ways:
1. Using Onyx selector but this actually does not take props (can't access fullReport.parentReportID), so we can't use this solution unfortunately https://expensify.slack.com/archives/C01GTK53T8Q/p1691169417094889
2. Use React.memo custom comparator, by default the memo will compare `prevProps.parentReportActions` with `nextProps.parentReportActions` (which will be different if any report action chane), we should instead compare `prevProps.parentReportActions[fullReport.parentReportActionID]` with `nextProps.parentReportActions[fullReport.parentReportActionID]`